cmake/FindPAPI.cmake

```diff
@@ -0,0 +1,45 @@
+# Try to find PAPI headers and libraries.
+#
+# Usage of this module as follows:
+#
+#     find_package(PAPI)
+#
+# Variables used by this module, they can change the default behaviour and need
+# to be set before calling find_package:
+#
+#  PAPI_PREFIX         Set this variable to the root installation of
+#                      libpapi if the module has problems finding the
+#                      proper installation path.
+#
+# Variables defined by this module:
+#
+#  PAPI_FOUND              System has PAPI libraries and headers
+#  PAPI_LIBRARIES          The PAPI library
+#  PAPI_INCLUDE_DIRS       The location of PAPI headers
+
+find_path(PAPI_PREFIX
+    NAMES include/papi.h
+)
```

Comment:
Just to finish this up. I installed papi out of the default folder instalation and the script wasn't able to find the proper location. I exported the enviroment variable PAPI_PREFIX with the proper location, but it didn't work. It worked when I modified the Cmakefile.txt setting the PAPI_PREFIX there. How did you do this?